controllers/users/index.js

```diff
@@ -112,9 +123,38 @@ const getCurrentUser = async (req, res) => {
   }
 };
 
+const avatarChange = async (req, res) => {
+  try {
```

Comment:
Ten blok try nie złapie błędu wewnątrz callbacku. Możesz to sprawdzić wpisując poniżej linijki 128 
`throw 'some error message`